src/widgets/Notebook.cpp

```diff
@@ -325,6 +330,48 @@ void Notebook::setAllowUserTabManagement(bool value)
     this->allowUserTabManagement_ = value;
 }
 
+bool Notebook::getShowTabs() const
+{
+    return this->showTabs_;
+}
+
+void Notebook::setShowTabs(bool value)
+{
+    this->showTabs_ = value;
+
+    this->performLayout();
+    for (auto &item : this->items_)
+    {
+        item.tab->setHidden(!value);
+    }
+
+    this->setShowAddButton(value);
+
+    // show a popup upon hiding tabs
+    if (!value && getSettings()->informOnTabVisibilityToggle.getValue())
+    {
+        QMessageBox msgBox;
+        msgBox.window()->setWindowTitle("Chatterino - hidden tabs");
+        msgBox.setText("You've just hidden your tabs");
+        msgBox.setInformativeText(
+            "You can toggle tabs by using the keyboard shortcut (Ctrl + U by "
+            "default) or right-clicking on tab area and selecting \"Toggle "
```

Comment:
```suggestion
            "default) or right-clicking on the tab area and selecting \"Toggle "
```
Super pedantic but I couldn't resist :grimacing:

Reply:
Nitpicks like this are supa cool and useful, no joke. Also pls approve once I fix this.